bench/run_benchmarks.py

```diff
@@ -112,7 +100,7 @@ def _wait_for_channel_start(self):
 
     def delete_ussd_channel(self, channelid):
         self.conn.request(
-            "DELETE", '/channels/%s' % channelid)
+            "DELETE", '/channels/%s' % channelid, json.dumps({}))
```

Comment:
I'm a little unsure why we need a body here? Since it's a delete request.

Reply:
The delete was failing and started working when I added this. Doesn't really make sense. Let me try without it and see.